### ISSUE TITLE
Bring UDUNITS Fortran interface to develop

### ISSUE THREAD

**Issue by bena-nasa** [GEOS-ESM/MAPL#2931]
For the work to do vertical regridding in ExtData in MAPL2 having UDUNITS would be useful. I've already confirmed that if I copy over the udunits2f directory from release/MAPL-v3 it builds with the appropriate update to cmake find udunits so there's no technical problem. Just making an issue here for the PR. Unfortunately since this was done on release/MAPL-v3 first we can't just merge this back...

So I'm not sure what the best way to do this would be to avoid conflicts. 

We could delete it in release/MAPL-v3, bring it to develop, then merge develop into release/MAPL-v3 as we have been doing?

**Comment by tclune** [comment]
If it had been a single commit, you could do a cherry-pick, but I think it will be impractical.   I think you have to just manually copy into place and deal with the merge conflicts - which will hopefully be of the trivial sort.

**Comment by mathomp4** [comment]
Yeah, probably just needs to be a "bring files over manually". Once you (and @darianboggs) have something, I can do a test merge into MAPL3 to make sure all will be happy.

And, if not, well, once it's in, I can just *force* Git to be happy by making sure MAPL3 stays MAPL3!

**Comment by bena-nasa** [comment]
cool, I'll do that

**Comment by darianboggs** [comment]
Closed by #2934
